recipes/breakpad/all/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class BreakpadTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            bin_path = os.path.join("bin", "test_package")
```

Comment:
This one does not need to use the binaries in the `bin` package folder?

Reply:
I think everything in `bin` are tools.

Reply:
Yup. I was asking whether breakpad has a handler binary, such as crashpad.

Reply:
Nop. Breakpad is easier to use since you just link the library to your app. No need to ship an extra handler.

Reply:
does not match this

Reply:
@prince-chrismc 
breakpad ships utility executables to examine stacktraces post-crash, but no `crashpad_handler`-like executable that must be executed in parallel with the *victim-executable*.

Reply:
Ahhhhhh thank you for explaining